src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,268 @@
+Parallel Integral Curve System
+==============================
+
+Within the VisIt infrastructure is the ability to generate integral curves:
+streamlines and pathlines. An integral curve is a curve that begins at a seed
+location X, and is tangent at every point in a vector field. An integral curve
+is computed by numerical integration of the seed location through the vector
+field. The image below shows fieldlines through the magnetic field of a
+core-collapse supernova simulation from the GenASiS code.
+
+IMAGE HERE
```

Comment:
missing image?

Reply:
Draft PR, I haven't added the images yet.